worksheet_C/readme.MD

```diff
@@ -0,0 +1,10 @@
+Worksheet C  
+A) The algorithm checks if there is a duplicate letter in the list by going through the list in order to see if a letter is repeated, if a letter has a duplicate the result will be true. 
+B) Because of the loop within the loop, so it has to check each item in the first list as well as each item in the second list so it will be N * N = N^2 
```

Comment:
There is no "first list" and "second list", only one list. Otherwise the "loop within the loop" explanation is correct.